python/ray/experimental/state/api.py

```diff
@@ -1168,7 +1168,7 @@ def get_log(
 
 
 def list_logs(
-    api_server_url: str = None,
```

Comment:
I made all the logs api to take in `address` rather than `api_server_url` to be consistent with the rest. 

I think we should consolidate this to either one of them. I am actually leaning towards `api_server_url` to be more accurate and specific. (`address` is fine with me since we call it out explicitly in most of the docs)

Reply:
Actually `address` means the GCS address, and api_server_url is the address of the API server. I think we should consolidate to `address` since we don't want to expose many URLs to users

Reply:
Are you saying we should auto resolve `gcs address` to `api server url`?

Reply:
yes. We should provide `address` (GCS server) and other URL should be all implementation detail